packages/deployments/contracts/deployments/goerli/RelayerProxyHub.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0xcF415Fe34CADaB2aD39C403A40F86216DbA20918",
```

Comment:
should we redeploy `RelayerProxyHub` contract for linea?

Reply:
Actually don't need to redeply, we can add/remove hub connectors.

Reply:
Lets revert it then

Reply:
reverted